### ISSUE TITLE
[2.0] Standard camera device profiles don't work

### ISSUE THREAD

**Issue by tonyespy** [edgexfoundry/device-camera-go#138]
The example device profiles in device-camera-go v2.0.0 are broken, as many of the [device resource names were re-formatted to use PascalCase](https://github.com/edgexfoundry/device-camera-go/pull/111) (aka CamelCase) however the `ProtocolDriver` implementation still uses lowercase with underscore characters for most ONVIF commands (there are some exceptions). 

Here's a snippet of the relevant code in driver.go:
```

switch req.DeviceResourceName {
    // ONVIF cases
    case "onvif_device_information":
        data, err = onvifClient.GetDeviceInformation()
	 if err != nil {
            d.lc.Error(err.Error())
            return responses, err
        }

       cv, err = sdkModel.NewCommandValue(reqs[i].DeviceResourceName, ...)
    case "onvif_profile_information":
        data, err = onvifClient.GetProfileInformation()
	 if err != nil {
	     d.lc.Error(err.Error())
             return responses, err
        }

        cv, err = sdkModel.NewCommandValue(reqs[i].DeviceResourceName, ...)
```

This can be tested natively by configuring a test device (Device001) to use any of the three example profiles, ensuring the credentials have been pushed to Vault (if required), and then issuing the following command:

```
$ curl -w %{http_code} http://localhost:59985/api/v2/device/name/Camera001/OnvifSnapshot
{"apiVersion": "v2", "message": "error reading DeviceResource OnvifSnapshot for Camera001 -\u003e Non-ONVIF command for camera without secondary client"
```

**Comment by cloudxxx8** [comment]
@tonyespy @lenny-intel may we just revert the PR?
https://github.com/edgexfoundry/device-camera-go/pull/111/files

**Comment by lenny-intel** [comment]
> @tonyespy @lenny-intel may we just revert the PR?

Naw, since we have to do a patch release either way we should do the missing code changes that were not in the PR.

**Comment by iain-anderson** [comment]
closed #140
